src/task.ts

```diff
@@ -83,6 +85,7 @@ export default class Task {
   }
 
   async instanceAt(name: string, address: string): Promise<Contract> {
+    await this.saveInInternalEVMState(address);
```

Comment:
```suggestion
```
We don't seem to need this on the getter (works for me without it, and seems like it would slow things down a lot).